.gitignore

```
@@ -22,4 +22,5 @@ artillery
 
 coverage.lcov
 .nyc_output
-.data
\ No newline at end of file
+.data
+coverage/
```

```suggestion
coverage/

```

Please could you add a new line to the end.
Good practice since some interpreters don't read the file properly without a final blank line